src/herder/TxSetFrame.cpp

```diff
@@ -369,6 +454,64 @@ TxSetFrame::previousLedgerHash() const
     return mPreviousLedgerHash;
 }
 
+size_t
+TxSetFrame::size(LedgerHeader const& lh) const
```

Comment:
I think this should take the _version_ as a parameter, not the ledgerHeader. At the call sites above, it reads as though it's somehow measuring the size of the ledgerHeader. I would also probably rename it, so like  `sizeInOpsOrTxs(uint32_t protocolVersion)`

Reply:
re: name, sound good.

For passing around an uint32, I normally try to avoid this as we lose type safety. I'll pick a better name.

Reply:
Mmmm, actually I can't find a better name. I guess we're dealing with one of the hardest problems in software engineering: "naming".

Goal when I used `size` was to keep this `size` function as the generic "size" that should be used for protocol version agnostic code (and the standard library uses that name).

I don't think I want to name it by listing all possible ways it can be implemented now or in the future `sizeInOpsOrTxsOrPoneys`.

The reason this question first came up was that the method signature implies that it's computing the size of the `LedgerHeader`, but I am not sure this is the case:
* if I wanted such a function it should be called `sizeOf` or something like that
* if I want to be explicit, I could call it something like `getTxSetSize`, but this is redundant (class name is `txSet` already)
* `size` is the method name used to return the size of containers in the standard library (you don't do `std::size(mylist)` but `mylist.size()`)

What do you think @graydon ?